arangod/Pregel/Status/ExecutionStatus.h

```diff
@@ -68,6 +68,7 @@ struct ExecutionTimings {
   Duration storing;
   // FIXME: just sum the times above?
   Duration total;
+  Duration fatal;
```

Comment:
I don't think we want to measure this duration

Reply:
Removed.